### ISSUE TITLE
关于ILI数据集

### ISSUE THREAD

**Issue by vaeaea** [plumprc/MTS-Mixers#1]
您好，请问可以分享一下关于ILI数据集的一些详细的参数设置吗？
我按照提供的默认参数运行后，得到的结果与文中相比差距很大

**Comment by vaeaea** [comment]
这是我目前使用的参数，我粗略的调整了几项后，发现mse依然维持在三点几，希望您能够帮助我改进它！

Args in experiment:
Namespace(activation='gelu', batch_size=32, c_out=7, checkpoints='./checkpoints/', d_ff=64, d_layers=1, d_model=32, data='custom', data_path='national_illness.csv', dec_in=7, devices='0,1,2,3', do_predict=False, dropout=0.05, e_layers=2, embed='timeF', embed_type=0, enc_in=7, fac_C=False, fac_T=True, factor=1, features='M', freq='t', gpu=0, individual=False, is_training=1, itr=1, label_len=1, learning_rate=0.01, loss='mse', lradj='type1', mat=0, model='MTSMixer', moving_avg=25, n_heads=1, norm=True, num_workers=10, output_attention=False, patience=3, pred_len=24, refine=False, rev=True, root_path='D:\\原型测试文件\\all_six_datasets\\illness', sampling=2, seg=20, seq_len=36, target='OT', test_flop=False, train_epochs=50, use_amp=False, use_gpu=True, use_multi_gpu=False)

**Comment by plumprc** [comment]
Hi,
Our results of ILI didn't include the use of `LayerNorm`. Please append `--norm` to close it.

**Comment by vaeaea** [comment]
@plumprc 
感谢你的帮助，另外我想请教一下关于layernorm的使用有什么技巧吗？还是说这只是通过实验得出的结果。
因为我之前了解到的情况是 layernorm可以有助于梯度稳定，但在这里不使用layernorm效果却会更好，我感到很好奇

**Comment by mqt922** [comment]
请问，它这个输入的数据是什么样式哒？

**Comment by vaeaea** [comment]
> 
（batch_size,seq_len,num_of_variable）

**Comment by vaeaea** [comment]
（batch_size,seq_len,num_of_variable） @mqt922

**Comment by plumprc** [comment]
> @plumprc 感谢你的帮助，另外我想请教一下关于layernorm的使用有什么技巧吗？还是说这只是通过实验得出的结果。 因为我之前了解到的情况是 layernorm可以有助于梯度稳定，但在这里不使用layernorm效果却会更好，我感到很好奇

这是实验得出的结果，我个人猜测是因为时间序列里的值和范围都很重要，因此可逆比起标准化要更加重要，如果网络参数比较少，加 LN 会破坏像均值、方差这些统计信息，所以预测结果会变差；但如果网络参数很多，像 PatchTST 那样，那么有些参数可能会自己去学习这种可逆性，或者说对齐（alignment）的关系，这样加了 LN 也不会有很大影响

**Comment by vaeaea** [comment]
> > @plumprc 感谢你的帮助，另外我想请教一下关于layernorm的使用有什么技巧吗？还是说这只是通过实验得出的结果。 因为我之前了解到的情况是 layernorm可以有助于梯度稳定，但在这里不使用layernorm效果却会更好，我感到很好奇
> 
> 这是实验得出的结果，我个人猜测是因为时间序列里的值和范围都很重要，因此可逆比起标准化要更加重要，如果网络参数比较少，加 LN 会破坏像均值、方差这些统计信息，所以预测结果会变差；但如果网络参数很多，像 PatchTST 那样，那么有些参数可能会自己去学习这种可逆性，或者说对齐（alignment）的关系，这样加了 LN 也不会有很大影响

好的，我明白了，谢谢你的分享！
